script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -572,34 +610,38 @@ Oh, how sinful it was――how foolish.
 [sha:50053fbf301454207b49cf7174f1b19ac30f0abf]{
 -- Page 684, Offset 16501.
 -- 彼はこの時、宇宙のすべてを論理的に明かしたいと、天上の主に願ったのだ。
-It was then that he swore to the heavens above that he would uncover the theory of the entire cosmos.
+// Is 天上の主 supposed to be e.g. heavenly father? Or is it fine to be secular/generic here? Doesn't seem to be a common phrase
+It was then that he swore to the heavens above that he would lay bare the theory that underpins the cosmos.
```

Comment:
It's probably supposed to be capital G, yes. Roa is a very pious man, after all.